mvc/Data/SeedData.cs

```diff
@@ -1,21 +1,52 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using FastMember;
+using LINQtoCSV;
+using Microsoft.Data.SqlClient;
 using Microsoft.EntityFrameworkCore;
 using SkladUcebnic.Models;
 
 namespace SkladUcebnic.Data
 {
     public static class SeedData
     {
+        public static async Task ImportCSVFile(SkladUcebnicContext dbContext)
+        {
+            var csvFile = @"wwwroot/csv/list.csv";
+            var cc = new CsvContext();
+
+            var csvFileDescription = new CsvFileDescription()
+            {
+                SeparatorChar = ',',
+                FirstLineHasColumnNames = true
+            };
+
+            var books = cc.Read<Book>(csvFile, csvFileDescription).ToList();
+
+            using (var sqlBulk = new SqlBulkCopy("Server=(localdb)\\mssqllocaldb;Database=SkladUcebnicContext-6e81ebcd-c2c2-415f-9c8d-81eddcc5f9ef;Trusted_Connection=True;MultipleActiveResultSets=true"))
```

Comment:
Preco potrebujeme `SqlBulkCopy`? Predpokladam, ze ma nejake performance vyhody, ale myslim ze to je pre nas zanedbatelne.

Ak tomu spravne rozumiem, tak knihy uz mame nacitane v `books`. Mali by sme ich vediet zapisat do DB pomocou `dbContext`u (rovnako ako v `SeedBooks`). To je idealne riesenie, kedze sa nam nebude miesat XY roznych 'pristupov na DB'.

Problematicke su tam dve veci:
- `ColumnMappings.Add` toto musi reflektovat `Book` model, to uz za nas ale riesi EntityFramework. Ak do buducna zmenime model, musime pamatam ze potrebujeme zmenit aj import.
- connection string priamo v kode - taketo veci je idealne tahat z nastaveni appky, tak ako to robime v `Program.cs`